### PR TITLE
Add format check for oap/velox

### PR DESCRIPTION
**Pull request by zhejiangxiaomai**
1. add format-check for oap/velox.
2. keep consistent with meta/velox. (same clang-format config and same version).

### CODE REVIEW

**Comment by PHILO-HE** [comment]
It seems `clang-format-12` is used in upstream velox. We should keep consistent, right?

File: .github/workflows/unittest.yml [hunk 1]
```diff
@@ -15,12 +15,16 @@ jobs:
...
+      - run: sudo apt-get install -y libz-dev python3-pip
+      - run: sudo pip install regex
       - name: Compile C++ unit tests
         run: |
           git submodule sync --recursive && git submodule update --init --recursive
           ./scripts/setup-ubuntu.sh 
           make release EXTRA_CMAKE_FLAGS="-DVELOX_ENABLE_ARROW=ON"
+      - name: Check formatting
+        run: |
+          ./scripts/check.py format commit
```
Comment (rui-mo) [inline .github/workflows/unittest.yml:30]: I notice Velox uses below command to trigger format check and fix, is this aligned with that?

```
make format-check
make format-fix
```

**Comment by rui-mo** [comment]
> It seems `clang-format-12` is used in upstream velox. We should keep consistent, right?

I agree. We'd better use the aligned version.

**Comment by zhouyuan** [comment]
it seems there are many format changes, are those due to local changes in OAP?

**Comment by zhejiangxiaomai** [comment]
> it seems there are many format changes, are those due to local changes in OAP?

I noticed, those code basically are added by us.

**Pull request merged**